import_map.jsonc

```diff
@@ -0,0 +1,8 @@
+{
+  "imports": {
+    "template/": "./src/",
+    "sift/": "https://deno.land/x/sift@0.4.2/",
+    "discordeno/": "https://raw.githubusercontent.com/discordeno/discordeno/main/",
```

Comment:
```suggestion
    "discordeno/": "https://deno.land/x/discordeno/mod.ts",
```

Reply:
I probably did the url wrong but this should ideally use deno.land/x/ instead of raw github